src/Evolution/Systems/Cce/AnalyticSolutions/BouncingBlackHole.hpp

```diff
@@ -38,28 +38,24 @@ struct BouncingBlackHole : public WorldtubeData {
     static constexpr Options::String help{
         "The coordinate distance of the gauge oscillation"};
     static type lower_bound() noexcept { return 0.0; }
-    static type default_value() noexcept { return 2.0; }
   };
   struct ExtractionRadius {
     using type = double;
     static constexpr Options::String help{
         "The extraction radius of the spherical solution"};
     static type lower_bound() noexcept { return 0.0; }
-    static type default_value() noexcept { return 20.0; }
```

Comment:
This might actually be a useful suggested value @moxcodes? (also `Amplitude` and `Period` to give an idea for useful values?)

Reply:
Yeah, I think the defaults for bouncing black hole should become suggested values -- there's some tuning that was done to choose values that give a careful test of the CCE system, but do not get so close to the black hole to cause problems for the null foliations.